react/features/salesforce/actions.js

```diff
@@ -18,9 +20,12 @@ import { SalesforceLinkDialog } from './components';
  */
 export function showSalesforceNotification() {
     return (dispatch: Object, getState: Function) => {
-        const { salesforceUrl } = getState()['features/base/config'];
+        const conference = getCurrentConference(getState());
+        const salesforceEnabled = isSalesforceEnabled(getState());
+        const skipNotification = !salesforceEnabled
```

Comment:
Why do you need the extra check if `isSalesforceEnabled` already checks if you are in a breakout room?

Reply:
Right, moved all the logic in one selector.